src/web/components/SharingPermission/BulkAddPermissions.tsx

```diff
@@ -2,31 +2,35 @@ import { useState } from 'react';
 import { FormProvider, useForm } from 'react-hook-form';
 
 import { ParticipantDTO } from '../../../api/entities/Participant';
+import { ParticipantTypeDTO } from '../../../api/entities/ParticipantType';
+import { AvailableParticipantDTO } from '../../../api/participantsRouter';
+import { Banner } from '../Core/Banner';
+import { Collapsible } from '../Core/Collapsible';
 import { FormStyledCheckbox } from '../Input/StyledCheckbox';
 import {
+  BulkAddPermissionsForm,
+  getCheckedParticipantTypeIds,
   getDefaultAdvertiserCheckboxState,
   getDefaultDataProviderCheckboxState,
   getDefaultDSPCheckboxState,
   getDefaultPublisherCheckboxState,
   getRecommendationMessageFromTypeNames,
+  hasUncheckedDefaultParticipantTypeRecommendation,
 } from './bulkAddPermissionsHelpers';
 
 import './BulkAddPermissions.scss';
 
 type BulkAddPermissionsProps = {
   participant: ParticipantDTO | null;
+  participantTypes: ParticipantTypeDTO[];
+  sharingParticipants: AvailableParticipantDTO[];
   onBulkAddSharingPermission: (selectedTypes: number[]) => Promise<void>;
 };
 
-export type BulkAddPermissionsForm = {
```

Comment:
Moved this to the helper file to avoid a circular dependency. Happy to change if there is a better way